repository.json

```diff
@@ -141,6 +141,39 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/sublimelsp/LSP-lua",
+			"name": "LSP-lua",
+			"releases": [
+				{
+					"date": "2020-09-20 14:07:53",
+					"platforms": [
+						"linux-x64"
+					],
+					"sublime_text": ">=3154",
+					"url": "https://github.com/sublimelsp/LSP-lua/releases/download/v0.0.3/LSP-lua_linux-x64.zip",
+					"version": "v0.0.3"
+				},
+				{
+					"date": "2020-09-20 14:07:53",
+					"platforms": [
+						"osx-x64"
+					],
+					"sublime_text": ">=3154",
+					"url": "https://github.com/sublimelsp/LSP-lua/releases/download/v0.0.3/LSP-lua_osx-x64.zip",
+					"version": "v0.0.3"
+				},
+				{
+					"date": "2020-09-20 14:07:53",
+					"platforms": [
+						"windows-x64"
+					],
+					"sublime_text": ">=3154",
```

Comment:
This should be `>=4000`, but I'm still figuring out a way to communicate that from the release all the way to here.